public/mobile-app/src/routes/+page.svelte

```diff
@@ -118,89 +152,92 @@ const registerWithAmi = async () => {
         p256dh: pushSubP256DH,
       },
     },
-    email: registerEmailValue,
+    email: registerEmailInputValue,
   }
-  console.log('isRegistered with AMI:', payload)
+  console.log('payload:', payload)
   isRegisteredWithAmi = true
-  registrationStatus = 'Registering...'
+  registrationStatus = "En cours d'enregistrement..."
 
   const response = await fetch(`${PUBLIC_API_URL}/notification/register`, {
     method: 'POST',
     body: JSON.stringify(payload),
   })
-  console.log('response from AMI:', response)
+  console.log('response:', response)
   isRegisteredWithAmi = false
   if (response.status < 400) {
-    registrationStatus = 'Done!'
-    userEmail = registerEmailValue
+    registrationStatus = 'Enregistrement réussi !'
+    userEmailState = registerEmailInputValue
+    userNotifications = []
```

Comment:
We probably want to call `retrieveNotifications` here so the list is refreshed as soon as the user is registered.